app-emulation/vice/vice-2.4.31.ebuild

```diff
@@ -0,0 +1,174 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit autotools eutils toolchain-funcs flag-o-matic
+
+DESCRIPTION="The Versatile Commodore 8-bit Emulator"
+HOMEPAGE="http://vice-emu.sourceforge.net/"
+SRC_URI="mirror://sourceforge/vice-emu/releases/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+IUSE="Xaw3d alsa ethernet ffmpeg fullscreen +gtk ipv6 lame nls oss png pulseaudio sdl +sdlsound threads vte zlib"
+
+# upstream says gtk3 and sdl2 shouldn't be exposed yet.
+#REQUIRED_USE="?? ( gtk2 gtk3 sdl )"
+REQUIRED_USE="?? ( gtk sdl )"
+
+GTK_COMMON="
+	x11-libs/pango
+	x11-libs/cairo"
+#	gtk3? (
+#		x11-libs/gtk+:3
+#		vte? ( x11-libs/vte:2.90 )
+#		${GTK_COMMON}
+#	)
+RDEPEND="
+	virtual/jpeg:0
+	virtual/opengl
+	media-libs/giflib
+	alsa? ( media-libs/alsa-lib )
+	pulseaudio? ( media-sound/pulseaudio )
+	sdlsound? ( media-libs/libsdl[sound] )
+	ethernet? (
+	    >=net-libs/libpcap-0.9.8
+	    >=net-libs/libnet-1.1.2.1:1.1
+	)
+	ffmpeg? ( virtual/ffmpeg )
+	lame? ( media-sound/lame )
+	nls? ( virtual/libintl )
+	png? ( media-libs/libpng:0 )
+	zlib? ( sys-libs/zlib )
+	sdl? (
+		media-libs/libsdl[joystick,video]
+	)
+	!sdl? (
+		fullscreen? (
+			x11-libs/libXrandr
+			x11-libs/libXxf86vm )
+		x11-libs/libX11
+		x11-libs/libXext
+		sys-libs/readline:0
```

Comment:
I've debated this on #gentoo-dev (I think you were present at the time) and my argument was that for packages you are linking against, the only reason another (sub)slot is likely to be added later is for a breaking ABI change or an soname change. I did challenge people to think up a different reason and no one did. In some rare cases, you may get an unneeded rebuild but I feel it's better to err on the side of caution.

I don't make a big deal about it because opinions differ and there is the preserved-rebuild safety net. I just thought I'd chime in after seeing your comment.